styles/FeaturedWork.module.css

```diff
@@ -0,0 +1,84 @@
+.featured-work {
+  margin-top: 2%;
+  margin-bottom: 2%;
```

Comment:
Margins are not usually percentages here for containers.. Using pixel or rem depends on if a responsive design is implemented.

Reply:
Fixed, thanks